shell/platform/darwin/ios/framework/Headers/FlutterAppDelegate.h

```diff
@@ -29,6 +29,15 @@ FLUTTER_EXPORT
 
 @property(strong, nonatomic) UIWindow* window;
 
+/**
+ * Handle StatusBar touches.
+ *
+ * If you want your Flutter views to respond to StatusBar touches, like scroll-to-top behavior, you
```

Comment:
Remove the "you" wording.

"Call this method from -[AppDelegate touchesBegan:withEvent:] to register for status bar touches...

Reply:
Done.  Too informal?